@xen-orchestra/backups/writers/DeltaReplicationWriter.js

```diff
@@ -80,6 +80,7 @@ exports.DeltaReplicationWriter = class DeltaReplicationWriter extends MixinRepli
   }
 
   async _transfer({ timestamp, deltaExport, sizeContainers }) {
+    const { ignoreReplicationTags } = this._settings
```

Comment:
This is hack property, feel free to name it `_warmMigration` to denotate it's internal.